### DISCUSSION TITLE
AutoFocusNextElement support tab key?

### DISCUSSION THREAD

**Discussion by luckypong** [unoplatform/uno#18058] [Q&A]
Currently it seems to only work with Enter key.
Tab key moves the focus sequentially.
Should I add TabIndex?

Are you thinking of integrating it with AutoFocusNextElement?

**Comment by MartinZikmund** [comment]
Indeed, `AutoFocusNextElement` does not support this currently, but I agree it would be useful. For now `TabIndex` should work, but feel free to create an issue to add this to autofocus
